.github/workflows/scanner-db-init-dump.yaml

```diff
@@ -0,0 +1,115 @@
+          git config --global --add safe.directory "$(pwd)"
+
+      - name: Build updater
+        run: |
+          make tag
+          make -C scanner bin/updater
+
+      - uses: actions/upload-artifact@v4
+        with:
+          name: updater
+          path: scanner/bin/updater
+
+  build-init-dump:
+    needs:
+      - build-updater
+    runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: registry.redhat.io/rhel8/postgresql-15
+        credentials:
+          username: ${{ secrets.RH_REGISTRY_USERNAME_RO }}
+          password: ${{ secrets.RH_REGISTRY_PASSWORD_RO }}
+        env:
+          POSTGRESQL_ADMIN_PASSWORD: scanner
+        # User "runner(1001)" owns the workspace directory.
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+          --user 1001
+        # Use workspace to ensure the volume permission is set to "runnner(1001)".
+        volumes:
+          - ${{ github.workspace }}:/var/lib/pgsql/data
+        ports:
+          - 5432:5432
+    env:
+      # TODO Hard-coded to "dev" while we don't have a matrix job to go over all
+      #      release branches.
+      version: dev
+    steps:
+      - uses: actions/download-artifact@v4
+        with:
+          name: updater
+          path: /usr/local/bin
+
+      - name: Install PostgreSQL 15 client
+        shell: bash
+        run: |
+          source /etc/os-release
+          echo "deb http://apt.postgresql.org/pub/repos/apt $VERSION_CODENAME-pgdg main" | sudo tee /etc/apt/sources.list.d/pgdg.list
+          curl -sSL https://www.postgresql.org/media/keys/ACCC4CF8.asc | sudo apt-key add -
+          sudo apt-get update
+          sudo apt-get install -y postgresql-client-15
+
+      - name: Run updater
+        run: |
+          chmod +x /usr/local/bin/updater
+          updater \
+              --vulns-url "https://storage.googleapis.com/scanner-v4-test/vulnerability-bundles/$version/vulns.json.zst" \
+              --db-conn 'host=localhost user=postgres database=postgres password=scanner'
+
+      - name: Run pg_dump
+        shell: bash
+        run: |
+          PGPASSWORD=scanner pg_dump \
+              -v \
+              -h localhost \
+              -p 5432 \
+              -U postgres \
+              --format=custom postgres \
+          | zstd -o "db-init-$version.dump.zst"
+
+      - uses: actions/upload-artifact@v4
+        with:
+          name: init-dump
+          path: db-init-*.dump.zst
+
+  upload-init-dump:
+    needs:
+      - build-init-dump
+    runs-on: ubuntu-latest
+    steps:
+      - name: Authenticate with Google Cloud
+        uses: google-github-actions/auth@v2
+        with:
+          credentials_json: ${{ secrets.GOOGLE_SA_CIRCLECI_SCANNER }}
+
+      - name: Set up Google Cloud SDK
+        uses: google-github-actions/setup-gcloud@v2
+
+      - uses: actions/download-artifact@v4
```

Comment:
nit: could add `name` and/or `path` so we're not grabbing all the artifacts

Reply:
I want to leave the door open to grab them all--my plan is to (maybe if we decide to continue with this) to build multiple bundle versions, one per release.

Reply:
It grabs all the artifacts in the job (e.g., the updater binary). Not a huge deal, especially if we're going to filter out what we're interacting with elsewhere in the job. Just something to keep in mind